### DISCUSSION TITLE
Welcome to s14clock Discussions!

### DISCUSSION THREAD

**Discussion by simpleavr** [simpleavr/s14clock#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ronolexa** [comment]
I received my kit Friday and got everything assembled and functioning.  I'm having a small issue:  the clock forgets it's WiFi programming.  It's on firmware 2.03.  It will run for hours just fine, then I'll find it displaying the SSID of it's internal AP.  Reboot brings it back to the same condition.  I connect to it, and assign the SSID and P/W of my WiFi and the unit connects and displays as it should.  But every few hours I need to repeat that process.  It seems that the WiFi information is forgotten and the unit falls back to it's original state.

Am I missing a step in WiFi setup, or is this a known problem?

Thanks...   ron

**Reply by simpleavr** [reply]
Hi Ron,
Haven't heard of similar issue before.
The WiFi library I used do reset the WiFi credentials if it cannot connect to WiFi after a few minutes.
Where there is a power outage, and the clock got reset, sometimes since the router is slow to recover (re-connects to the internet), my clock unit would have it's WiFi credentitals reset and I will have to re-enter the setup / WiFi info again.
The clock also does not connects to the internet or use WiFi continously, it will connect to Wifi every few hours only to sychronize the time. I would suspect that when it try to sychronize, it is not able to connect and after a few tries, it resets the WiFi setting and went back to the setup mode.

I recommend you to try the following;

- upgrade to the latest firmware, you can try the beta firmware (WiFi part is the same though) to make sure clock is most up-to-date.
- try to place your clock unit closer to your router and observe for a day or two. The S2 module might have trouble due to distance and / or how busy your WiFi is.
- check your router setup, I am no expert on those and it could happen that your router have certain requirements for client device to connect, etc.
- it might be worthwhile to check also your usb cable and power source.

[reload firmware here](https://csimpleavr.github.io/s14clock/install.html)

If issue cannot be resolve with above, please post again and I will try to investigate further based on new observations.
Thanks...Chris.

**Comment by ronolexa** [comment]
Thanks Chris,

I believe the issue may be due to the way my mesh WiFi system works:  I 
have a Netgear Deco system running in the house.  I also have the 
generally unused AP supplied as part of the cable modem bringing my 
Internet service.

I connected to the Modem AP, and all the problems disappeared.  Put it 
back on the Deco AP, and the problems returned within 3 hours.  It's now 
been running solid on the modem AP for 30 hours.

Not sure why the Deco acts differently, but this experience does help me 
out on another project.  I have another ESP based project that exhibits 
similar loss of WiFi, although it corrects with a reboot.  I always 
thought it was an issue with the ESP or software, but now I'm thinking 
it's due to the Deco AP.  I'll need to swap it over to the modem 
directly and see if the issue disappears.

Thanks again...   ron

On 2024-06-02 12:28, simpleavr wrote:

> Hi Ron,
> Haven't heard of similar issue before.
> The WiFi library I used do reset the WiFi credentials if it cannot 
> connect to WiFi after a few minutes.
> Where there is a power outage, and the clock got reset, sometimes since 
> the router is slow to recover (re-connects to the internet), my clock 
> unit would have it's WiFi credentitals reset and I will have to 
> re-enter the setup / WiFi info again.
> The clock also does not connects to the internet or use WiFi 
> continously, it will connect to Wifi every few hours only to sychronize 
> the time. I would suspect that when it try to sychronize, it is not 
> able to connect and after a few tries, it resets the WiFi setting and 
> went back to the setup mode.
> 
> I recommend you to try the following;
> 
> * upgrade to the latest firmware, you can try the beta firmware (WiFi 
> part is the same though) to make sure clock is most up-to-date.
> * try to place your clock unit closer to your router and observe for a 
> day or two. The S2 module might have trouble due to distance and / or 
> how busy your WiFi is.
> * check your router setup, I am no expert on those and it could happen 
> that your router have certain requirements for client device to 
> connect, etc.
> * it might be worthwhile to check also your usb cable and power source.
> 
> reload firmware here [1]
> 
> If issue cannot be resolve with above, please post again and I will try 
> to investigate further based on new observations.
> Thanks...Chris.
> 
> --
> Reply to this email directly, view it on GitHub [2], or unsubscribe 
> [3].
> You are receiving this because you commented.Message ID: 
> ***@***.***>



Links:
------
[1] https://csimpleavr.github.io/s14clock/install.html
[2] 
https://github.com/simpleavr/s14clock/discussions/1#discussioncomment-9639471
[3]

**Reply by simpleavr** [reply]
Glad it worked out.
Thanks for posting the findings here as it might help others w/ similar issues.
